src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -293,14 +293,14 @@ private static ScheduledThreadPoolExecutor initSyncExecutor()
 
     private volatile double crcCheckChance;
 
-    public static Iterable<SSTableReader> selectOnlyBigTableReaders(Iterable<? extends SSTableReader> readers)
+    public static Iterable<? extends SSTableReader> selectOnlyBigTableReaders(Iterable<? extends SSTableReader> readers)
```

Comment:
This would be better as
`public static <T extends SSTableReader> Iterable<T> selectOnlyBigTableReaders(Iterable<T> readers)`